packages/markdown-docx/test/data/ciceroMark/ternary-operator-in-text.json

```diff
@@ -0,0 +1 @@
+{"$class":"org.accordproject.commonmark.Document","xmlns":"http://commonmark.org/xml/1.0","nodes":[{"$class":"org.accordproject.commonmark.Heading","level":"2","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"Try TemplateMark"}]},{"$class":"org.accordproject.commonmark.Paragraph","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"Just trying "},{"$class":"org.accordproject.commonmark.Strong","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"few things "},{"$class":"org.accordproject.commonmark.Emph","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"here"}]}]},{"$class":"org.accordproject.commonmark.Softbreak"},{"$class":"org.accordproject.commonmark.Strong","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"hello"}]},{"$class":"org.accordproject.commonmark.Text","text":" "},{"$class":"org.accordproject.commonmark.Strong","nodes":[{"$class":"org.accordproject.commonmark.Text","text":"there"}]}]}]}
```

Comment:
You aren't parsing the "ternary operator in text". You are parsing text nodes that only have a space. One question here - do you have an idea when MS Word interprets something as space and when as a softbreak? Shouldn't this space have been a softbreak instead? If no, could you send the markdown file you parsed this CiceroMark from?

Reply:
The markdown file is 

**strong**_italic_

![Screenshot from 2021-07-05 16-51-03](https://user-images.githubusercontent.com/59891164/124463939-34d11680-ddb1-11eb-9f23-ef1bd3cb2a0d.png)

Reply:
> You aren't parsing the "ternary operator in text". You are parsing text nodes that only have a space. One question here - do you have an idea when MS Word interprets something as space and when as a softbreak? Shouldn't this space have been a softbreak instead? If no, could you send the markdown file you parsed this CiceroMark from?

I am not sure but I think `xmljs` might not consider space string as elements. Otherwise, it won't throw `cannot read property 0 of undefined`(AFAIR).

Reply:
> You aren't parsing the "ternary operator in text". You are parsing text nodes that only have a space.

Said this with respect to the file name of the test case.

Reply:
Yeah, I got that. I was just explaining the reason for the space thing which might be due to `xmljs`. I will rename the file.
I mistakenly quoteed the complete statement instead of the line
>One question here - do you have an idea when MS Word interprets something as space and when as a softbreak? Shouldn't this space have been a softbreak instead?

Reply:
> I am not sure but I think xmljs might not consider space string as elements. Otherwise, it won't throw cannot read property 0 of undefined(AFAIR).

I don't think so XMLJS is screwing up here. It's the template mark dingus because it should have interpreted that space as softbreak.

Anyway, if there are cases like this, then it makes sense to have that ternary operator.

Reply:
@algomaster99 
Made the push. Is the current name correct?

Reply:
Sounds good.